docs/source/workflow_inputs.rst

```diff
@@ -148,7 +148,7 @@ Building construction is entered in ``/HPXML/Building/BuildingDetails/BuildingSu
   =============================================================  ========  =========  =================================  ========  ========  =======================================================================
 
   .. [#] ResidentialFacilityType choices are "single-family detached", "single-family attached", or "apartment unit".
-         For ENERGY STAR, "single-family detached" may only be used for SF versions and "apartment unit" may only be used for MF versions; "single-family attached" may be used for all versions.
+         For ENERGY STAR, "single-family detached" and "single-family attached" may only be used for SF versions and "apartment unit" may only be used for MF versions. 
```

Comment:
The language was revised to reflect the changes listed in the [highlights](https://www.energystar.gov/sites/default/files/2025-01/Revision%2005%20Highlights.pdf).
"_The National Program Requirements eligibility has been revised to reflect the change that townhouses are no longer
eligible to participate in MFNC and all townhouse-specific requirements have been removed from all documents._"

Reply:
I have a small concern that there still might be building types considered single-family attached that are allowed to use MFNC, but I'm not really sure. Let's ask EPA. If we don't hear back from EPA, I think we should leave it how it is to be flexible for now.

Reply:
Based on EPA's response, I think we should go forward with your proposed change. So to continue with it, I think we need to:
1) Call this out in the changelog as a breaking change.
2) Remove [this code](https://github.com/NREL/OpenStudio-ERI/blob/es_new_sfnh_mfnc/rulesets/resources/ES_ZERHruleset.rb#L128-L141), which no longer applies.
3) Update any sample files or tests that may break because of this change.